extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/MSQControllerTask.java

```diff
@@ -147,6 +149,24 @@ public String getType()
     return TYPE;
   }
 
+  @Override
+  public String getLabel() {
+    if (isIngestion(querySpec)) {
+      if (((DataSourceMSQDestination) querySpec.getDestination()).isReplaceTimeChunks()) {
+        return "msq_index_batch_replace";
```

Comment:
I feel this is to easy to break. Its much better we add a label to the destination in case that is what you are looking for and then use destinaton.getLablel() here. 
WDYT ?

Reply:
I am unclear about how this would change in autocompaction.